CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ endif()
 # project name
 project(SFML VERSION 3.0.0)
 
-set(VERSION_IS_RELEASE OFF)
+set(SFML_VERSION_IS_RELEASE OFF)
```

Comment:
What made you want to rename this variable?

Reply:
For simplicity, so the variable can be handled with `cmakedefine01` without any ceremony.